gammapy/conftest.py

```diff
@@ -12,6 +12,10 @@
 ## from the list of packages for which version numbers are displayed
 ## when running the tests ... this was added in Astropy 1.0
 try:
+    # Removing h5py for now to get rid of this travis-ci fail
+    # https://travis-ci.org/gammapy/gammapy/jobs/86565582#L1074
+    del PYTEST_HEADER_MODULES['h5py']
```

Comment:
@astrofrog - Thanks.

But surely there should be a way or at least workaround to run Astropy and affiliated package tests even if h5py is broken. Having a dependency which isn't even used in most affiliated package break their continuous integration is bad, no?

If this is really not possible at the moment, I think we should open a ticket to change the setup.
Should this be in the Astropy or Astropy-helpers tracker?